src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -26,6 +26,26 @@ export interface ConnectorInvoice {
   type: "received";
 }
 
+// @Todo: extend missing properties accordingly
+export interface ConnectorTransaction {
+  custom_records: Record<number, number>;
+  fee: number;
+  keysend: boolean;
+  memo: string;
+  payment_hash: {
+    data: number[];
+    type: string;
+  };
+  payment_preimage: string;
+  r_hash: {
+    data: number[];
+    type: string;
+  };
+  timestamp: number;
+  type: string;
```

Comment:
what is the type field for?

Reply:
I just implemented this one (as all other fields) because it is part of the response.

Reply:
Outgoing and Incoming are not shown in a single list, they have separate lists with tabs. Is this intended to be changed? How should I proceed, just remove it?

Reply:
maybe it could be explicitly sent to "sent" instead of a string for now.